doc/source/setup-helm.rst

```diff
@@ -28,7 +28,9 @@ cluster. At the terminal, enter:
 
    .. code:: bash
 
-      helm init
+      kubectl --namespace kube-system create sa tiller
```

Comment:
Perhaps use the more verbose command for user understanding: `kubectl create serviceaccount --namespace kube-system tiller`